CDP4Addin/Addin.cs

```diff
@@ -92,6 +94,11 @@ public class Addin : COMAddin
         /// </summary>
         private IOfficeApplicationWrapper officeApplicationWrapper;
 
+        /// <summary>
+        /// Gets or sets the instance of the <see cref="AppSettingsService"/> used to get application settings.
+        /// </summary>
+        private IAppSettingsService<AddinSettings> AppSettingsService { get; } = new AppSettingsService<AddinSettings>();
```

Comment:
This entire thing is not need here. Delete completely. The `AppSettingsService` is needed in the `CDP4AddinBootstrapper` wherre it uses another `PluginLoader()` just like in the IME.

Reply:
Logic has been moved to bootstrapper